nengo_bones/templates/setup.py.template

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+import imp
+import io
+import os
+import sys
+
+try:
+    from setuptools import find_packages, setup
+except ImportError:
+    raise ImportError(
+        "'setuptools' is required but not installed. To install it, "
+        "follow the instructions at "
+        "https://pip.pypa.io/en/stable/installing/#installing-with-get-pip-py")
+
+
+def read(*filenames, **kwargs):
+    encoding = kwargs.get("encoding", "utf-8")
+    sep = kwargs.get("sep", "\n")
+    buf = []
+    for filename in filenames:
+        with io.open(filename, encoding=encoding) as f:
+            buf.append(f.read())
+    return sep.join(buf)
+
+
+root = os.path.dirname(os.path.realpath(__file__))
+version_module = imp.load_source(
+    "version", os.path.join(root, "{{ PACKAGE }}", "version.py"))
+testing = "test" in sys.argv or "pytest" in sys.argv
+
+install_requires = [
+]
+docs_require = [
+    "sphinx",
+    "nengo_sphinx_theme",
+]
+optional_requires = [
+]
+tests_require = [
+    "pytest>=3.6",
+]
+
+
+setup(
+    name="{{ REPO NAME }}",
```

Comment:
Is the setup.py template actually being used anywhere?

Reply:
No, I just copied all these files to this folder before starting the changes in this PR.  We should probably just delete them though, they'll be in the git history if we want to pull them out later for something.

Reply:
Done (there were 5 of these files from what I could see).